cwe-377.py

```diff
@@ -0,0 +1,9 @@
+from tempfile import mktemp
+
+def write_results(results):
```

Comment:
(pylint-misplaced-future) __future__ import is not the first non docstring statement
(pylint-missing-function-docstring) Missing function or method docstring